app/assets/javascripts/two-pane.js

```diff
@@ -28,6 +28,13 @@ $(document).on('click', "a[href$='new'].new_two_pane", function(e) {
   }
 });
 
+$(document).on('click', "a[href$='edit'].edit_two_pane", function(e) {
```

Comment:
is this related ? this can effect many pages? (just wondering if it should be its own pr)